tests/pytests/test_short_read.py

```diff
@@ -597,21 +597,27 @@ def runShortRead(env, data, total_len, expected_index):
 seed = str(time.time())
 random.seed(seed)
 
-def downloadFile(file_name):
+def downloadFile(env, file_name):
     path = os.path.join(REDISEARCH_CACHE_DIR, file_name)
     path_dir = os.path.dirname(path)
     os.makedirs(path_dir, exist_ok=True)
     if not os.path.exists(path):
         subprocess.run(["wget", "--no-check-certificate", BASE_RDBS_URL + file_name, "-O", path, "-q"])
         if os.path.splitext(path)[-1] == '.zip':
-            return unzip(path, path_dir)
+            res = unzip(path, path_dir)
+            env.assertTrue(res, message='Failed to unzip ' + path)
+            return res
         else:
-            return os.path.exists(path) and os.path.getsize(path) > 0
+            exists = os.path.exists(path)
+            non_zero_size = os.path.getsize(path) > 0
```

Comment:
it is unsafe to split this condition, as you cannot call `os.path.getsize(path)` if the path does not exist